Automattic/woocommerce-payments
# feat: enable tokenized ECE by default

Fixes https://github.com/Automattic/woocommerce-payments/issues/9724

#### Changes proposed in this Pull Request









Enabling the tokenized ECE by default.
It can be disabled via a filter, until we remove the feature flag:
```php
add_filter( 'pre_option__wcpay_feature_tokenized_cart_ece', function ( $v ) {
    // ensures that once the next version of WCPay is released, this filter can be automatically ignored
    if ( version_compare( WCPAY_VERSION_NUMBER, '8.9.1', '<' ) ) {
        return '0';
    }
 
    return $v;
}, 100 );
```

> [!IMPORTANT]
>  I am planning to merge this after the release of WooPayments 8.8.0, so it can land in WooPayments 8.9.0.

#### Testing instructions





- Ensure you have GooglePay/ApplePay enabled in WooCommerce > Settings > Payments > WooPayments 
- Go to the product page for a simple product
- Click on the GooglePay/ApplePay button
- In your browser's network tab, you should see calls to `/wp-json/wc/store/v1/cart/add-item` (for example)
- Apply the filter above (e.g.: via Code Snippets)
- Refresh the product page from earlier
- Click on the GooglePay/ApplePay button
- In your browser's network tab, you should now see calls to `/?wc-ajax=wcpay_add_to_cart` (for example) and no longer to the Store API

-------------------


**Post merge**
